events/beam-robotics.html

```diff
@@ -0,0 +1,154 @@
+---
+layout: default
+title: Robotics BEAM page
+permalink: /beam-robotics
+---
+<div class="beam-top-section">
+  <div class="grid-col">
+    <h1 class="title page-title"> Robotics Fun with BEAM </h1>
+    <p class="title text-2x no-margin">
+        <a class="tla-link" href="https://beam.ucla.edu/" target="_blank" rel="noopener noreferrer">
+       BEAM at UCLA</a> X ACM Teach LA
+   </p>
+  </div>
+</div>
+
+<p>
+    We partnered with <a class="tla-link" href="https://beam.ucla.edu/" target= "_blank" rel="noopener noreferrer"><b>BEAM at UCLA</b></a> to teach 4th and 5th graders some cool <b>robotics</b>! We have <b>4 weeks</b> of <b>hour long lessons</b> and you can follow along what we did here!
+</p>
+
+<hr class="divider" />
+
+<div class="secondary-top-section">
+    <p class="title text-2x no-margin">
+        <b>Setup</b>
+    </p>
+    <p>
+        In these lessons, we will learn how to code <a class="tla-link" href="https://www.makeblock.com/mbot" target="_blank" rel= "noopener noreferrer" ><b>mBots</b></a>! Before these lessons, you may want to <b>acquire an <a class="tla-link" href="https://www.makeblock.com/mbot" target="_blank" rel= "noopener noreferrer" >mBot</b></a> which you will learn to program, and <a class="tla-link" href="https://mblock.makeblock.com/en-us/download/" target="_blank" rel= "noopener noreferrer" ><b>download mBlock</b></a> which will be where you code your robot.
+    </p>
+    <div class ="centered-button-container">
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fa fa-link"></span> mBot purchase Link</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fa fa-link"></span> mBlock Download Guide</a>
+    </div>
+</div>
+
+<hr class="divider" />
+
+<div class="secondary-top-section">
+    <p class="title text-2x no-margin">
+        <b>Lessons</b>
+    </p>
+</div>
+
+<div class="event-section">
+    <div class="grid-col">
+      <img class="img-responsive" src="{{site.baseurl}}/img/beam-robotics/monkey.svg" alt="a monkey from a game you'll code in our week 1 lesson, sitting under a bananna tree"/>
```

Comment:
ty for all the alts :)

can we steal a wider image from the figma? the image is extra long/not as well sized as the other sections (lmk if you need help with this!)

id prefer if the image height was scaled up to the text, rather than extra space in terms of the image! (hopefully that makes sense)